frontend/src/metabase/actions/containers/ActionCreator/tests/ActionCreator-QueryActions.unit.spec.ts

```diff
@@ -45,6 +47,67 @@ describe("ActionCreator > Query Actions", () => {
         screen.getByRole("button", { name: "Action settings" }),
       ).toBeInTheDocument();
     });
+
+    describe("Save Modal", () => {
+      const originalAce = window.ace;
+
+      beforeAll(() => {
+        window.ace = {
+          edit: jest.fn(),
+        };
+      });
+      afterAll(() => {
+        window.ace = originalAce;
+      });
```

Comment:
I find that adding `id={ACE_ELEMENT_ID}` to the `textarea` in the mock `NativeQueryEditor` can make the test work without having to mock ace's `edit` function manually here. I think that might be preferable since we basically double-mock here. One for `NativeQueryEditor` and one for `ace`s `edit` which I think should be taken care of by the mocked `NativeQueryEditor` instead.

What do you think?

https://github.com/metabase/metabase/blob/7b47197d561ed51a8765b4c7d75e8fea23001a4c/frontend/src/metabase/query_builder/components/__mocks__/NativeQueryEditor.jsx#L14

Reply:
it worked, all other tests passed too, thanks!